person.rb

```diff
@@ -1,13 +1,20 @@
+require_relative 'nameable'
+
 # This a Person class
-class Person
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
```

Comment:
- [ ] [Optional] You do not need the `super` method in this `initialize` method because the `Nameable` class has no initialize method. The mere fact that it inherits from Nameable, it will have the same attributes and methods as Nameable. 👍

Reply:
Dear @Gambit142 thanks for the comment. I was never sure if it's required or not, but the linter complained when I left the supper() method out. That's why.